src/main/java/de/hysky/skyblocker/config/configs/SlayersConfig.java

```diff
@@ -57,4 +63,17 @@ public static class VampireSlayer {
         @SerialEntry
         public int maniaUpdateFrequency = 5;
     }
+
+    public enum HighlightSlayerEntities {
```

Comment:
This enum should go above `EndermanSlayer` class since it appears first in the config. This is a way to keep code clean and avoid merge conflicts. (This caused one merge conflict.)